### PR TITLE
Update config.rb

### PR DESCRIPTION
**Pull request by coolguru**

### CODE REVIEW

File: stack-audit-cloud-threats/services/config.rb [hunk 1]
```diff
@@ -171,9 +171,9 @@
     'route_table' => [],
     'route_table_association' => [],
     'subnet' => [],
-    'instance' => [],
+    'instance' => ['public_ip_address'],
     'iam_instance_profile' => [],
     'role' => [],
-    'policy' => []
+    'policy' => ['policy_arn']
```
Comment (armaneous) [inline stack-audit-cloud-threats/services/config.rb]: `policy_arn` doesn't really change, but I guess it's good practice to do explicitly state the trigger.

File: stack-audit-cloud-threats/services/config.rb [hunk 2]
```diff
@@ -87,7 +87,7 @@
     }
   }
   QUERY
-  meta_rule_node_triggers ({'internet_gateway' => ['relates_to'], 'route' => [], 'route_table' => [], 'route_table_association' => [], 'instance' => [], 'iam_instance_profile' => [], 'role' => [] })
+  meta_rule_node_triggers ({'internet_gateway' => ['relates_to'], 'route' => [], 'route_table' => [], 'route_table_association' => [], 'iam_instance_profile' => [], 'role' => [] })
```
Comment (armaneous) [inline stack-audit-cloud-threats/services/config.rb]: Can this be formatted like the one below? That way each node type is on its own line and easier to follow.

Comment (armaneous) [reply stack-audit-cloud-threats/services/config.rb]: Also, `instance` is a correct trigger, though I think you also need to add `public_ip_address` property trigger to it like below.

Comment (coolguru) [reply stack-audit-cloud-threats/services/config.rb]: there is bug in rule-eval which is making uid substitution incorrectly. No point to even trigger this with public_ip_address property change.

File: stack-audit-cloud-threats/services/config.rb [hunk 3]
```diff
@@ -87,7 +87,13 @@
     }
   }
   QUERY
-  meta_rule_node_triggers ({'internet_gateway' => ['relates_to'], 'route' => [], 'route_table' => [], 'route_table_association' => [], 'instance' => [], 'iam_instance_profile' => [], 'role' => [] })
+  meta_rule_node_triggers ({
+    'internet_gateway' => ['relates_to'], 
+    'route' => [], 'route_table' => [],
```
Comment (armaneous) [inline stack-audit-cloud-threats/services/config.rb]: Still need `'instance' => ['public_up_address']` as well as moving the above two into separate lines.

Comment (coolguru) [inline stack-audit-cloud-threats/services/config.rb]: CON-768 to add instance when we fix pla-6680

File: stack-audit-cloud-threats/services/config.rb (hunk 1 above)
Comment (scvenema) [reply stack-audit-cloud-threats/services/config.rb]: Yes, with the `policy_arn` qualifier, this `policy` trigger will probably never run. So, yes, we could probably omit that entire trigger.  But unless this is significant computational overhead, I'd much rather document expectations here than omit them and have someone later wondering why that predicate wasn't included in the trigger.

**Comment by scvenema** [comment]
Starting at line 90, the trigger rules should instead be the following? :
  meta_rule_node_triggers ({
    'internet_gateway' => ['relates_to'], 
    'route' => [],
    'route_table' => [], 
    'route_table_association' => [],
    'subnet' => [],
    'instance' => [],
    'iam_instance_profile' => [], 
    'role' => [],
    'policy' => ['policy_arn']
  })

**Pull request merged**